tools/consensus_merge.cwl

```diff
@@ -0,0 +1,85 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: consensus_merge
+doc: >-
+  This tool will run a Python script that merges somatic calls from several callers
+      into a single consensus VCF
+requirements:
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/consensus-merge:1.0.0'
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: ${ return inputs.ram * 1000 }
+    coresMin: $(inputs.cores)
+  - class: ShellCommandRequirement
+
+baseCommand: [/usr/bin/consensus_merge.py]
+
+inputs:
+  strelka2_vcf:
+    type: File
+    inputBinding:
+      position: 1
+      prefix: '--strelka2_vcf'
+    secondaryFiles: .tbi
```

Comment:
It seems technically this is ok, but on cwltool, when buried in a subwf of a subwf, using:
`secondaryFiles: .tbi` instead of 
`secondaryFiles: ['.tbi']` trips up the validator. Please change the rest to fit that format

Reply:
✅